File: stack/typescript/MainStack.ts
import { Stack } from "./Stack";


const numberStack = new Stack<number>(10);

console.log("\nTamaño de Stack: ", numberStack.getSize())
console.log("Esta vacio el Stack? ", numberStack.isEmpty())
console.log("Esta lleno el Stack? ", numberStack.isFull())
numberStack.showStack()

// Insertando datos en el Stack
numberStack.push(1)
numberStack.push(2)
console.log("\nTamaño de Stack: ", numberStack.getSize())
console.log("Esta vacio el Stack? ", numberStack.isEmpty())
console.log("Esta lleno el Stack? ", numberStack.isFull())
numberStack.showStack()

// Insertando datos en el Stack
numberStack.push(4)
numberStack.push(7)
console.log("\nTamaño de Stack: ", numberStack.getSize())
console.log("Esta vacio el Stack? ", numberStack.isEmpty())
console.log("Esta lleno el Stack? ", numberStack.isFull())
numberStack.showStack()

numberStack.pop()
numberStack.showStack()

// Insertando datos en el Stack
numberStack.push(5)
numberStack.push(6)
numberStack.push(8)
numberStack.push(9)
numberStack.showStack()

File: stack/python/MainStack.py
from Stack import Stack

stack = Stack(10)

print(stack.get_size())
print(stack.is_empty())
print(stack.is_full())

stack.push(1)
stack.push(2)
stack.push(4)
stack.push(6)
print(stack.get_size())
print(stack.is_empty())
print(stack.is_full())

stack.pop()
stack.push(9)
stack.push(11)
stack.push(7)
stack.push(14)
stack.push(21)
print(stack.get_size())
print(stack.is_empty())
print(stack.is_full())

stack.push(57)
stack.push(85)
stack.push(91)
stack.push(101)